README.md

```diff
@@ -12,12 +14,16 @@ The Open Neural Network Exchange implementation in MLIR (http://onnx.ai/onnx-mli
 | amd64-Windows | [![Build Status](https://dev.azure.com/onnx-pipelines/onnx/_apis/build/status/MLIR-Windows-CI?branchName=main)](https://dev.azure.com/onnx-pipelines/onnx/_build/latest?definitionId=9&branchName=main)             |
 | amd64-macOS   | [![Build Status](https://github.com/onnx/onnx-mlir/actions/workflows/macos-amd64-build.yml/badge.svg)](https://github.com/onnx/onnx-mlir/actions/workflows/macos-amd64-build.yml)             |
 
+This project contributes an ONNX Dialect that can be integrated in other projects, a compiler interfaces that lower ONNX graphs into MLIR files/LLVM bytecodes/C & Java libraries, an `onnx-mlir` driver to perform these lowering, and a python/C/C++/Java runtime environment.
+
 ## Setting up ONNX-MLIR using Prebuilt Containers
 
-The preferred approach to using and developing ONNX-MLIR is to used Docker Images and Containers, as getting the proper code dependences may be tricky on some systems. Our instructions on using ONNX-MLIR with dockers are [here](docs/Docker.md).
+The preferred approach to using and developing ONNX-MLIR is to use Docker Images and Containers, as getting the proper code dependences may be tricky on some systems. Our instructions on using ONNX-MLIR with dockers are [here](docs/Docker.md).
 
 ## Setting up ONNX-MLIR directly
 
+ONNX-MLIR runs natively on Linux, OSX, and Windows.
```

Comment:
This seems incomplete. What are the instruction to setup the project *natively* (without using a virtualized environment) for the project on those platforms.  Not every contributor will want (or be able to) use a virtualized environment and so we should write instructions for all the native env. supported.